packages/pact-consumer-tests/src/generators/init/files/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Why do we generate a package.json in the project files? Should we not add these as dependencies?